### ISSUE TITLE
Warn about no automain

### ISSUE THREAD

**Issue by matthewp** [stealjs/steal#1418]
Add a warning in dev mode if no main is provided and nothing is loaded after a certain time period.

**Comment by matthewp** [comment]
Be very careful with this one, none of these should trigger a warning:

```html
<script src="/path/to/steal.js"></script>
<script>
  System.import("foo");
</script>
```

```html
<script src="/path/to/steal.js"></script>
<script>
  steal.import("foo");
</script>
```

```html
<script src="/path/to/steal.js">
  import "foo";
</script>
```

```html
<script src="/path/to/steal.js"></script>
<script type="steal-module">
  import "foo";
</script>
```

I think the best bet would be to wrap loader.import to see if something is loaded post startup or not.

**Comment by matthewp** [comment]
Fixed by #1441
